runtime/perfetto/db.go

```diff
@@ -120,14 +130,16 @@ func Open(ctx context.Context, fname string) (*DB, error) {
 	}
 
 	t := &DB{
+		opts:            opts,
 		fname:           fname,
 		db:              db,
 		replicaNumCache: cache,
 	}
 
-	const initTables = `
+	initTables := `
 -- Trace data.
 CREATE TABLE IF NOT EXISTS traces (
+	insertion_time INTEGER NOT NULL,
 	app TEXT NOT NULL,
 	version TEXT NOT NULL,
 	events TEXT NOT NULL
```

Comment:
For future consideration: I just noticed the events field. Is that some text based rendering of the trace data? Would we be better off preserving the structure inherent in the trace entries by storing each event as a separate row with columns corresponding to event timestamp, etc.